ratis-server-api/src/main/java/org/apache/ratis/server/DivisionProperties.java

```diff
@@ -47,6 +47,14 @@ default int maxRpcTimeoutMs() {
     return maxRpcTimeout().toIntExact(TimeUnit.MILLISECONDS);
   }
 
+  /** @return the bound of leader lease timeout */
+  TimeDuration leaderLeaseTimeout();
+
+  /** @return the bound of leader lease timeout in milliseconds */
+  default int leaderLeaseTimeoutMs() {
+    return leaderLeaseTimeout().toIntExact(TimeUnit.MILLISECONDS);
+  }
```

Comment:
This should be internal use only.  Let's not add it `ratis-server-api`?

Reply:
Done.